android/BOINC/app/src/main/java/edu/berkeley/boinc/PrefsFragment.java

```diff
@@ -385,6 +434,7 @@ private void setupDialogButtons(final PrefsListItemWrapper item, final Dialog di
 		confirm.setOnClickListener(new OnClickListener() {
 			@Override
 			public void onClick(View v) {
+				// Sliders
```

Comment:
Unfortunately this whole block is still incorrectly indented. The `//Sliders` comment is the only correct one. All the `if` constructs that follow are aligned to `setupDialogButtons()`, not to `onClick()`. Can you please have another look? Thanks.

Reply:
First let's have a working version, then I'll work on the indentation later.

Reply:
Fixed.